elfpy/__init__.py

```diff
@@ -2,6 +2,7 @@
 
 import shutil
 import logging
+from typing import Dict
```

Comment:
If think if you add `from __future__ import annotations` you might be able to use `dict` instead of `Dict`. The latter is being deprecated in favor of the former in later versions of python.